templates/default/opensshd.conf.erb

```diff
@@ -157,7 +157,7 @@ ClientAliveInterval <%= @node['ssh-hardening']['ssh']['server']['client_alive_in
 ClientAliveCountMax <%= @node['ssh-hardening']['ssh']['server']['client_alive_count'] %>
 
 # Disable tunneling
-PermitTunnel no
+PermitTunnel <%= ((@node['ssh-hardening']['ssh']['server']['permit_tunnel']) ? 'yes' : 'no' ) %>
```

Comment:
from the man page:
```
     PermitTunnel
             Specifies whether tun(4) device forwarding is allowed.  The argu-
             ment must be yes, point-to-point (layer 3), ethernet (layer 2),
             or no.  Specifying yes permits both point-to-point and ethernet.
             The default is no.
```

So, we have more values then yes and no. How do we want to handle that? Mapping true/false to yes/no and everything else would be passed as-as, @chris-rock @atomic111 @runningman84 your views?